blueprints/ability-test/index.js

```diff
@@ -6,6 +6,8 @@ module.exports = {
   _getTestStyle: function() {
     if ('ember-cli-mocha' in this.project.addonPackages) {
       return 'mocha';
+    } else if (false) {
```

Comment:
At this point, I am not sure what to do without some digging. I imagine someone who has contributed to this addon before might better know how to determine when to set this line to true.

But that is it. After that, the new tests ought to be generated.

Reply:
I've seen in [Ember Source](https://github.com/emberjs/ember.js/blob/master/blueprints/test-framework-detector.js) that it may work like this:
```suggestion
    } else if ('ember-qunit' in this.project.addonPackages) {
```
I don't know how to create tests for blueprints, also. I think that ember-cli doesn't support this right now. But after the change I may be able to test the code manually if you like.